### ISSUE TITLE
Liveness Probes Fail for Non-AL4 Service Dependencies

### ISSUE THREAD

**Issue by srhalstead** [CybercentreCanada/assemblyline-v4-service#484]
**Issue:**

Attempts to create a custom service that leverages the REST API of a vendor-issued non-AL4 container are currently blocked because liveness checks on the vendor container fail.

**Further detail:**

I'm prototyping a custom service that interfaces with a vendor-supplied container via its REST API. I'm starting the vendor container as a dependency in the service manifest as shown below:

```
dependencies:
  vendor_service:
    container:
      <docker-config>
```

The service pod and its dependent pod start as expected but attempts to connect from the service pod to the vendor service fail. The address provided in the `vendor_service_host` environment variable is not resolvable. 
Further digging showed that the dependent pod (which runs the vendor-supplied image) is running but not marked as `READY`. Running `kubectl describe` on the dependent pod showed the following:
```
Warning  Unhealthy  3m6s (x3 over 13m)   kubelet            Liveness probe failed: /usr/bin/python3: Error while finding module specification for 'assemblyline_v4_service.healthz' (ModuleNotFoundError: No module named 'assemblyline_v4_service')
```
The vendor-supplied image was not developed for Assemblyline so the AL4 libraries are not present.

According to this [discussion on Discord](https://discord.com/channels/908084610158714900/908717441771794472/1050736501148893244), the capability should support non-AL4 containers but there was a false assumption made about pods in the liveness checks.

**Comment by cccs-rs** [comment]
Related feature ticket: https://cccs.atlassian.net/browse/AL-1976

**Comment by cccs-rs** [comment]
This should be resolved in [4.3.1.7](https://github.com/CybercentreCanada/assemblyline/releases/tag/v4.3.1.stable7)
